server/src/routes/api/user.mjs

```diff
@@ -27,21 +27,43 @@ import * as stateManagement from '../../stateManagement.mjs';
 
 // --- Route Handlers ---
 
-// POST /api/v1/user
+// PUT /api/v1/user
 // Expected body: N/A
 function addUser(req, res) {
-  // Generate a unique userId for this user (no vanity userIds, at least for now)
-  const userId = uuidv4();
+  let userId = req.params.userId;
```

Comment:
Let's do a null check against req.params to be safe

Reply:
Taken care